src/schema.js

```diff
@@ -258,6 +258,42 @@ const unflagPostSchema = Joi.object({
 			.required()
 	}
 })
+
+const updateUserSchema = Joi.object({
+	params: {
+		id: Joi.number()
+			.required()
+	},
+	body: {
+		firstName: Joi.string()
+			.alphanum()
+			.min(3)
+			.max(30)
+			.required(),
+
+		lastName: Joi.string()
+			.alphanum()
+			.min(3)
+			.max(30)
+			.required(),
+
+		email: Joi.string()
```

Comment:
@itzgeebee their email is our main way of identifying them, and they were invited using this same address by an admin so not really an option for them to change

Reply:
Oh okay. noted